androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/home/HomeFooter.kt

```diff
@@ -0,0 +1,157 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.home
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.material.*
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.extension.placeholder
+import co.nimblehq.avishek.phong.kmmic.android.ui.common.HorizontalPagerIndicator
+import co.nimblehq.avishek.phong.kmmic.android.ui.common.NextCircleButton
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun HomeFooter(
+    pagerState: PagerState,
+    pageCount: Int,
+    isLoading: Boolean,
+    surveyUiModel: SurveyUiModel?,
+    modifier: Modifier = Modifier,
+    onSurveyClick: (SurveyUiModel?) -> Unit,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 20.dp)
+    ) {
+        if (isLoading) {
+            HomeFooterPlaceholderContent()
+        } else {
+            HomeFooterContent(
+                pagerState = pagerState,
+                pageCount = pageCount,
+                surveyUiModel = surveyUiModel,
+                onSurveyClick = onSurveyClick
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+private fun HomeFooterContent(
+    pagerState: PagerState,
+    pageCount: Int,
+    surveyUiModel: SurveyUiModel?,
+    onSurveyClick: (SurveyUiModel?) -> Unit,
+) {
+    HorizontalPagerIndicator(
+        pagerState = pagerState,
+        pageCount = pageCount,
+        activeColor = White,
+        inactiveColor = White.copy(alpha = 0.20f),
+        spacing = 5.dp
+    )
+    Spacer(modifier = Modifier.height(26.dp))
+    Crossfade(targetState = surveyUiModel?.title) {
+        Text(
+            text = it.orEmpty(),
+            color = White,
+            style = typography.h5,
+            maxLines = 2,
+            overflow = TextOverflow.Ellipsis
+        )
+    }
+    Spacer(modifier = Modifier.height(10.dp))
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
```

Comment:
In Figma, they are bottom aligned. 🙏 
```suggestion
        verticalAlignment = Alignment.Bottom
```